playbooks/roles/tarsnap/README.md

```diff
@@ -2,8 +2,8 @@
 
 - An ansible role for operating tarsnap backups
 
-The role downloads sources for, verifies the gpg-encrypted sha signature, compiles, and installs [Tarsnap].  
-**Batteries included**: [tarsnapper], cron job, shell wrapper, and [logrotate] policy.
+The role downloads sources for, verifies the gpg-encrypted sha signature, compiles, and installs [Tarsnap].
+**Batteries included**: cron job, shell wrapper, and [logrotate] policy.
 
 ## Requirements
```

Comment:
@taranjeet This section contains a leftover broken link:

```
See [role variables](#Variables)
```

Could you just remove that part completely?